src/irmin/tree_intf.ml

```diff
@@ -115,30 +121,6 @@ module type S = sig
 
   (** {1 Manipulating Subtrees} *)
 
-  (** Operations on {{!node} lazy tree nodes}. *)
```

Comment:
Removed intentionally?

Reply:
Is this useful to expose this submodule?  Happy to keep it if really needed :-)

Reply:
We added this to support better external iterators when optimising the Tezos snapshot export (https://github.com/mirage/irmin/pull/1022). Perhaps not necessary now that we have better / faster iterators in the Tree API.

Reply:
sounds good to keep it then and we can just remove `bindings` (as `list` now also returns the subtrees)

Reply:
Happy to drop it in favour of just providing better iterators directly; we can always re-add if we find a need later. Have updated the changelog accordingly.